CheckoutSDK/Classes/TransactionParams.swift

```diff
@@ -14,25 +14,28 @@ public struct TransactionParams {
     public var email: String
     /// Your business' public key found at https://dashboard.paystack.com/keys
     public var publicKey: String
-    /// The customer's first name
     public var firstName: String?
-    /// The customer's last name
     public var lastName: String?
-    /// The customer's phone number
     public var phone: String?
+    public var reference: String?
     public var plan: String?
     public var invoiceLimit: Int?
     public var subAccount: String?
     public var transactionCharge: Int?
     public var bearer: String?
+    public var metaData: String?
+    public var quantity: Int?
+    public var splitCode: String?
+    public var label: String?
     public var currency: Currency?
```

Comment:
Should this be an optional parameter ? Can we have a transaction without a currency ?

Reply:
Yeah, currency is is optional. If no currency is set, the backend uses the business' default currency